ourteam.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>African-Marketplace</title>
+  <link href="https://fonts.googleapis.com/css2?family=Caveat&family=Raleway:ital@1&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style/index.css">
```

Comment:
Does the
```
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Do a good job with the mobile view loading in at correct magnification here?

 I remember encountering a weird kind of bug with that before.

Looking good.